joomla/joomla-cms
# [5.3] Optional content.prepare on text fields

Pull Request for Issue #42824 .

### Summary of Changes
Add a field param to the text field plugin to allow content.prepare plugins such as mailcloak to process the field


### Testing Instructions
Create a text field for an article
Enter an email address into the article
Check the output of the article and see if the email address is cloaked


### Actual result BEFORE applying this Pull Request
Email address is plain text


### Expected result AFTER applying this Pull Request
Email address is plain text
Change the field value in the text field plugin to Yes - the email address is cloaked
Change the field value in the field to No - the email address is not cloaked
Change the field value in the field to Yes - the email address is cloaked

![image](https://github.com/user-attachments/assets/efdd00e9-67b4-4209-aec7-ac4dc1c8a26a)

![image](https://github.com/user-attachments/assets/2333f14d-3f48-453a-aafa-11b272c99dd7)

This is fully b/c as the content.prepare is not enabled by default

### Link to documentations
Please select:
